npm/react/package.json

```diff
@@ -14,10 +14,14 @@
     "test": "yarn cy:run",
     "watch": "yarn build --watch --watch.exclude ./dist/**/*"
   },
+  "dependencies": {
+    "semver": "^7.3.2"
```

Comment:
Noticed this is getting bundled in the rollup output. It's not a huge amount of code, but since the `cli` package has `semver` listed as a dependency we could mark this package as "external" and save some bundle size.

Reply:
Had this thought too, but what about people using via npm? I think we should make the adapters as self-contained as possible?

Also, since we do a deep import, the bundle size barely changes - we only grab the `major` function.

Reply:
If it's listed as a dependency, then users who download it via npm will have semver downloaded for them. I'm not saying remove it from `packageJson.dependencies`, just have it added to `rollupConfig.externals`.

Reply:
Right, gotcha.